.prow.yaml

```diff
@@ -32,10 +32,10 @@ presubmits:
           resources:
             requests:
               memory: "512Mi"
-              cpu: "200m"
+              cpu: "1.5"
```

Comment:
"1.5" will get converted to "1500m" when deployed, this may result in argocd complaining (not sure), but let's just set it to "1500m" to be consistent with the api spec, same with the other decimal values.